### DISCUSSION TITLE
Default kubernetes-mixin dashboard show no data for cpu usage

### DISCUSSION THREAD

**Discussion by bespokeeagle** [prometheus-operator/kube-prometheus#1712] [Q&A]
Hi everyone 

i have a kube-prometheus setup release 0.9, but some of my kubernetes-mixin default dashboards show no data for a lot of cpu related information.
for example 

![image](https://user-images.githubusercontent.com/49434547/162416086-084a4e62-1b53-4cda-9940-aed1ae02e713.png)

i've spent several hours looking for a solution but to no avail yet.
Any help is highly appreciated

**Comment by bryanasdev000** [comment]
Related to https://github.com/prometheus-operator/kube-prometheus/discussions/1694.

**Reply by bespokeeagle** [reply]
@bryanasdev000 yeah, i just havent been able to find an answer

**Reply by bryanasdev000** [reply]
I think the same steps for troubleshooting in #1694 maybe used here:

https://github.com/prometheus-operator/kube-prometheus/discussions/1694#discussioncomment-2449811 and https://github.com/prometheus-operator/kube-prometheus/discussions/1694#discussioncomment-2559110
